draft-ietf-ace-oscore-profile.xml

```diff
@@ -645,66 +735,149 @@
+        </t>
+
+        <t>
+          The columns of the registry are:
+        </t>
+
+        <t>
+          <list style="hanging">
+            <t hangText='name'>
+              This is a descriptive name that enables easier reference to the item.  It is not used in the encoding.
+            </t>
+            <t hangText='label'>
+              The value to be used to identify this algorithm.
+              Key map labels MUST be unique.
+              The label can be a positive integer, a negative integer or a string.
+              Integer values between 0 and 255 and strings of length 1 are designated as Standards Track Document required.
+              Integer values from 256 to 65535 and strings of length 2 are designated as Specification Required.
+              Integer values of greater than 65535 and strings of length greater than 2 are designated as expert review.
+              Integer values less than -65536 are marked as private use.
+            </t>
+            <t hangText='CBOR Type'>
+              This field contains the CBOR type for the field.
+            </t>
+            <t hangText='registry'>
+              This field denotes the registry that values may come from, if one exists.
+            </t>
+            <t hangText='description'>
+              This field contains a brief description for the field.
+            </t>
+            <t hangText='specification'>
+              This contains a pointer to the public specification for the field if one exists
+            </t>
+          </list>
+        </t>
+
+        <t>
+          This registry will be initially populated by the values in <xref target="table-key-labels"/>.
+          The specification column for all of these entries will be this document.
+        </t>
+
+      </section>
+
+      <section title="CWT Confirmation Methods Registry">
+
+        <t>The following registration is done for the CWT Confirmation Methods Registry following the procedure specified in section 7.2.1 of <xref target="I-D.ietf-ace-cwt-proof-of-possession"/>:</t>
+
+        <t><?rfc subcompact="yes"?><list style='symbols'>
+          <t>Confirmation Method Name: "OSCORE_Security_Context"</t>
+
+          <t>Confirmation Method Description: OSCORE_Security_Context carrying the OSCORE Security Context parameters</t>
+
+          <t>Confirmation Key: TBD (value between 4 and 255)</t>
+
+          <t>Confirmation Value Type(s): map</t>
+        
+          <t>Change Controller: IESG</t>
+
+          <t>Specification Document(s): <xref target="oscore-sec-ctx"/> of &SELF;</t>
+        </list></t>
+
+      </section>
+
+
+      <section title="JWT Confirmation Methods Registry">
+
+        <t>The following registration is done for the JWT Confirmation Methods Registry following the procedure specified in section 6.2.1 of <xref target="RFC7800"/>:</t>
+
+        <t><?rfc subcompact="yes"?><list style='symbols'>
+          <t>Confirmation Method Value: "osc"</t>
+
```

Comment:
Did you specify what the type of the json field would be anyplace?

Reply:
No, I could not see that in the registration template...

Reply:
It needs to be stated in the document not the registry.

Reply:
Ok I tried to do that, see dc23146 ... Although I am not 100% sure this is the best way to describe the object.